backend/main.py

```
@@ -2,203 +2,381 @@
+                # Если all_pages=True — отдаем всё, иначе — только первую страницу
+                result_pages = pages if all_pages else pages[:1]
+
+                drawing_meta["image"] = {
+                    "base64": result_pages,
+                    "total_pages": len(pages),  # Оставляем реальное кол-во страниц для информации
+                    "content_type": "image/png"
+                }
+            else:
+                drawing_meta["image"] = None
+
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)